supervisor/utils/log_format.py

```diff
@@ -12,14 +12,14 @@
 )
 
 
-def format_message(message: str) -> str:
-    """Return a formated message if it's known."""
+async def format_message(message: str) -> str:
+    """Return a formatted message if it's known."""
     try:
         match = RE_BIND_FAILED.match(message)
         if match:
             return f"Port '{match.group(1)}' is already in use by something else on the host."
     except TypeError as err:
         _LOGGER.error("The type of message is not a string - %s", err)
-        capture_exception(err)
+        await async_capture_exception(err)
```

Comment:
Hm, it seems rather annoying that we have to convert so much to co-routines just because of that odd chance that the type here is wrong. In sentry there is not a single entry from this point.

And actually, checking code I think it is not even possible to get this ValueError :thinking: 

I'd prefer if we just drop this capture. We could convert it into a `_LOGGER.critical` (which ends up in Sentry, not sure how asyncio friendly this is exactly :wink: ), and/or don't handle the exception at all, and let it "bubble up".